apps/bfd-server/bfd-server-war/src/main/java/gov/cms/bfd/server/war/r4/providers/BeneficiaryTransformerV2.java

```diff
@@ -265,6 +264,8 @@ private static Patient transform(Beneficiary beneficiary, RequestHeaders request
           new DateTimeType(
               TransformerUtilsV2.convertToDate(beneficiary.getBeneficiaryDateOfDeath().get()),
               TemporalPrecisionEnum.DAY));
+    } else {
+      patient.setActive(true);
```

Comment:
I would also, based on the spec:
`patient.setDeceased(false)`

Reply:
done.